src/bridge.py

```diff
@@ -32,25 +32,36 @@ def cb_sim_pose(data):
     clarke_poses = data.poses[0]
     clarke_position = clarke_poses.position
     clarke_orientation = clarke_poses.orientation
-    
-    dr_msg = DeadReckonReport()
-    dr_msg.x = clarke_position.x
-    dr_msg.y = clarke_position.y
-    dr_msg.z = clarke_position.z
-
-    depth = clarke_position.z
-    
-    x, y, z, w = clarke_orientation.x, clarke_orientation.y, clarke_orientation.z, clarke_orientation.w
-    
-    roll, pitch, yaw = tf.transformations.euler_from_quaternion([x, y, z, w])
-    
-    dr_msg.roll = roll
-    dr_msg.pitch = pitch
-    dr_msg.yaw = yaw
-    
-    pub_dvl_deadreckon.publish(dr_msg)
-    pub_depth_sensor.publish(depth)
-
+    pub_state_x.publish(clarke_position.x + 3)
+    pub_state_y.publish(clarke_position.y)
+    pub_state_z.publish(clarke_position.z)
+
+    pose = Pose()
+    pose.position.x = clarke_position.x + 3
+    pose.position.y = clarke_position.y
+    pose.position.z = clarke_position.z
+    pose.orientation = clarke_orientation
+    pub_pose.publish(pose)
+
+    q = np.array([clarke_orientation.x, clarke_orientation.y, clarke_orientation.z, clarke_orientation.w])
+    conversion = transformations.euler_from_quaternion(q, 'rxyz')
+    theta_x = conversion[0]
+    theta_y = conversion[1]
+    theta_z = conversion[2]
+
+    angles = np.array([theta_x, theta_y, theta_z])*DEG_PER_RAD
+
+    for i in range(3):
+            if angles[i] - euler[i] > ANGLE_CHANGE_TOL:
+                euler[i] = angles[i] - 360
+            elif euler[i] - angles[i] > ANGLE_CHANGE_TOL:
+                euler[i] = angles[i] + 360
+            else:
+                euler[i] = angles[i]
+
+    pub_state_theta_x.publish(euler[0])
+    pub_state_theta_y.publish(euler[1])
+    pub_state_theta_z.publish(euler[2])
```

Comment:
Would you prefer the main branch to be the one that bypasses state estimation and a secondary one that doesn't bypass or the other way around?

Reply:
whichever one works should be main branch, in the future ideally we want to not need to bypass at all